### PR TITLE
Update server.es.yml (Allow welcome post...)

### PR DESCRIPTION
**Pull request by SidVal**
>FEATURE: Allow welcome post to be delayed via a site setting.
Added: https://github.com/discourse/discourse-narrative-bot/commit/70f87390f0b123ebca552513cf843fc5f9e674c7#diff-aeb8651199334c1cb831e884ff85773a

### CODE REVIEW

File: config/locales/server.es.yml [hunk 1]
```diff
@@ -374,6 +375,7 @@ es:
           ¿Sabes que puedes agregar una encuesta en cualquier mensaje? Intenta usando el botón de engranaje <img src="/images/font-awesome-gear.png" width="16" height="16"> en el editor para **armar una encuesta**.
         not_found: |-
           Whoops! No hubo ninguna encuesta en tu respuesta.
+
           Usa el ícono de engranaje: <img src="/images/font-awesome-gear.png" width="16" height="16"> en el editor, o bien copia y pega esta encuesta en tu próxima respuesta:
```
Comment (SidVal) [inline config/locales/server.es.yml]: Check this out @ZogStriP 
Here (in english) is happened too: https://github.com/discourse/discourse-narrative-bot/blob/70f87390f0b123ebca552513cf843fc5f9e674c7/config/locales/server.en.yml#L404

The line 404 (in server.en.yml) is highlighted in green when It's text (nor a text-label for translation).
Can you check it? 

(I hope you can understand what I mean) :blush:

**Review by SidVal — COMMENTED** [review]
Check this out @ZogStriP 
Here (in english) is happened too: https://github.com/discourse/discourse-narrative-bot/blob/70f87390f0b123ebca552513cf843fc5f9e674c7/config/locales/server.en.yml#L404

The line 404 (in server.en.yml) is highlighted in green when It's text (nor a text-label for translation).
Can you check it? 

(I hope you can understand what I mean) :blush:

**Comment by ZogStriP** [comment]
I'm sorry, I'm not sure what you mean...

**Comment by SidVal** [comment]
green-line

Maybe with a screen?

**Comment by ZogStriP** [comment]
Oh, that's because GitHub's syntaxic coloration isn't handling yaml multilines properly.

**Comment by SidVal** [comment]
Ah well, then there is no problem with this.
I had the doubt, since in the spanish's file of translation, also has that line in green. 

I was afraid it would not work properly. :+1:

**Pull request merged**